kyaml/fn/runtime/container/container_test.go

```diff
@@ -68,7 +70,8 @@ metadata:
 		},
 
 		{
-			name: "storage_mounts",
+			name:          "storage_mounts",
+			skipOnWindows: true, // the fieldpaths are different on Windows
```

Comment:
Alternatively, I could separate the tests two different files (one for Windows and one for everything else), and have different "expected" values for each

Reply:
It was more than the separator, I think the whole absolute path looks different. But using the filepath package is a good idea, I'm going to try using filepath.Join or potentially filepath.Abs if that doesn't work.

Reply:
It worked! Thanks for the suggestion.